script/oracles/config/ethereum.json

```diff
@@ -1,41 +1,28 @@
 [
-    {
-        "name": "WETH (wstETH)",
```

Comment:
Why removing those configs ?

Reply:
else it will deploy those oracles as well